src/drivers/intel/cavs/ipc.c

```diff
@@ -120,7 +120,7 @@ static void ipc_irq_handler(void *arg)
 
 static enum task_state ipc_platform_do_cmd(void *data)
 {
-#if CAVS_VERSION < CAVS_VERSION_2_0
+#if !defined CONFIG_SUECREEK
```

Comment:
In general `#if !CONFIG_SUECREEK` is enough.

Reply:
Please change to !CONFIG instead of !defined (to have it consistent across codebase).